posthog/temporal/data_modeling/run_workflow.py

```diff
@@ -284,6 +291,13 @@ async def handle_model_ready(model: ModelNode, team_id: int, queue: asyncio.Queu
         await queue.put(QueueMessage(status=ModelStatus.FAILED, label=model.label, error=str(err)))
     except Exception as err:
         await logger.aexception("Failed to materialize model %s due to error: %s", model.label, str(err))
+
+        if "job_id" in locals():
```

Comment:
any reason you're only marking the job as failed when in this block? Shouldn't we also report failures for the other exception types?

Reply:
Ah, yes, these exception changes got merged in from another change, but needs to be manually updated here as well.